precalculus/exercises/outcomes/PR/PR1/generator.sage

```diff
@@ -64,7 +64,9 @@ class Generator(BaseGenerator):
             face = "downwards"
             extrema = "maximum"
         
-        
+        form = choice(["standard","vertex"])
+        if form=="vertex":
```

Comment:
```suggestion
        if choice(True,False):  # use vertex form instead
```

Reply:
take or leave this suggestion

Reply:
A bit late, but I actually think my code is more descriptive and like it a bit better.